### DISCUSSION TITLE
How to turn off this feature?

### DISCUSSION THREAD

**Discussion by ghost** [apache/netbeans#5519] [Q&A] [answered]
I don't like it!

![Capture](https://user-images.githubusercontent.com/119163165/219478809-35f8ac07-2c10-4933-bd7b-048324da648e.PNG)

**Comment by neilcsmith-net** [comment, marked as answer]
View menu. Toggle off inline hints.

**Reply by ghost** [reply]
What an annoying feature! I don't need it!
